api/api.go

```diff
@@ -217,4 +217,6 @@ func configHandlers(e *echo.Echo) {
 
 	e.GET("/apps/:app/rules", appRules)
 	e.POST("/apps/:app/sync", appForceSyncRule)
+
+	e.GET("/jobs/:app/rules", jobRules)
```

Comment:
should this be `e.GET("/jobs/:job/rules", jobRules)`?

Reply:
I think there are no tests for this layer (not just this change, but the front api layer)